resources/BundleInfo.plist.in

```diff
@@ -326,6 +326,19 @@
 					<string>dxf</string>
 				</array>
 			</dict>
+			<!-- X -->
+			<dict>
+				<key>CFBundleTypeName</key>
+				<string>DirectX</string>
+				<key>CFBundleTypeRole</key>
+				<string>Viewer</string>
+				<key>LSIsAppleDefaultForType</key>
+				<true/>
+				<key>CFBundleTypeExtensions</key>
+				<array>
+					<string>x</string>
+				</array>
+			</dict>
```

Comment:
I completely forgot about this file, it misses many file formats and it should be configured depending on which plugins are enabled. Should I open an issue ?

Reply:
Ideally, it should be handled by the plugin framework instead of being its own thing.

Reply:
Yeah you're right, it should be handled by plugins. Or maybe we shouldn't care about this file here and just handle it in the superbuild.  
Yes, please open an issue, we need to clarify that at some point.

Reply:
https://github.com/f3d-app/f3d/issues/1135